block/manager.go

```diff
@@ -375,14 +374,6 @@ func (m *Manager) UpdateTargetHeight(h uint64) {
 
 // ValidateConfigWithRollappParams checks the configuration params are consistent with the params in the dymint state (e.g. DA and version)
 func (m *Manager) ValidateConfigWithRollappParams() error {
-	drsVersion, err := version.GetDRSVersion()
```

Comment:
where it's gonna be validated now?
if the DRS changed in the rollapp params (through the RDK)
where it's validated against local version?

Reply:
its validated in the begin blocker function of the app https://github.com/dymensionxyz/rollapp-evm/pull/393. this way it allows using a different version when upgrade is required, but if using wrong version it will fail in begin blocker